object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java

```diff
@@ -175,6 +138,40 @@ private void handleEnsuresCalledMethodVarArgs(
     }
   }
 
+  @Override
+  public void accumulate(Node node, TransferResult<CFValue, CFStore> result, String... values) {
+    super.accumulate(node, result, values);
+    if (exceptionalStores == null) {
+      return;
+    }
+    List<String> valuesAsList = Arrays.asList(values);
+    // If dataflow has already recorded information about the target, fetch it and integrate
+    // it into the list of values in the new annotation.
+    Receiver target = FlowExpressions.internalReprOf(atypefactory, node);
+    if (CFAbstractStore.canInsertReceiver(target)) {
+      CFValue flowValue = result.getRegularStore().getValue(target);
+      if (flowValue != null) {
+        Set<AnnotationMirror> flowAnnos = flowValue.getAnnotations();
+        assert flowAnnos.size() <= 1;
+        for (AnnotationMirror anno : flowAnnos) {
+          if (typeFactory.isAccumulatorAnnotation(anno)) {
+            List<String> oldFlowValues =
+                ValueCheckerUtils.getValueOfAnnotationWithStringArgument(anno);
+            if (oldFlowValues != null) {
+              // valuesAsList cannot have its length changed -- it is backed by an
+              // array.  getValueOfAnnotationWithStringArgument returns a new,
+              // modifiable list.
+              oldFlowValues.addAll(valuesAsList);
+              valuesAsList = oldFlowValues;
+            }
+          }
+        }
+      }
+    }
+    AnnotationMirror newAnno = typeFactory.createAccumulatorAnnotation(valuesAsList);
+    exceptionalStores.values().stream().forEach(s -> s.insertValue(target, newAnno));
```

Comment:
Is the eventual fix to have the called methods checker itself propagate to exceptional stores in this manner?  Or is there still some discussion to be had as to whether that is the right thing to do?